v0/destinations/lambda/transform.js

```diff
@@ -0,0 +1,56 @@
+const { getErrorRespEvents } = require("../../util");
+
+function process(event) {
+  const { destination } = event;
+  const { invocationType, clientContext, lambda } = destination.Config;
```

Comment:
Don't we need validation for these fields?

Reply:
Added necessary validations and tests in the latest commit